packages/webview_flutter/webview_flutter_android/README.md

```diff
@@ -11,6 +11,21 @@ normally. This package will be automatically included in your app when you do.
 
 This package uses [pigeon][3] to generate the communication layer between Flutter and the host platform (Android). The communication interface is defined in the `pigeons/android_webview.dart` file. After editing the communication interface regenerate the communication layer by running the `./generatePigeons.sh` shell script.
 
+Due to [flutter/flutter#97744](https://github.com/flutter/flutter/issues/97744), the generated test
+pigeon file needs to update one of its imports to properly work with `mockito`.
```

Comment:
s/needs to update of its imports/needs one of its imports updated/ (the file isn't going to do the update :) )